package.json

```diff
@@ -21,9 +21,11 @@
     "svgo": "^2.5.0"
   },
   "scripts": {
+    "lint": "eslint --report-unused-disable-directives --ignore-path .gitignore .",
+    "fix": "npm run lint -- --fix",
     "build": "gulp clean-build --production",
     "start": "sirv build --port 8080 --dev --no-clear",
     "dev": "gulp dev",
-    "test": "gulp build --production"
+    "test": "npm run lint && gulp build --production"
```

Comment:
Maybe group lint, fix, test together?

Reply:
Done. BTW isn't the `production` flag moot?

Reply:
🤷‍♂️

Reply:
Yup, I confirmed it, the `--production` flag is moot since a long time.